suite-common/metadata-types/src/metadataTypes.ts

```diff
@@ -185,15 +187,18 @@ export interface WalletLabels {
     walletLabel?: string;
 }
 
-export type Labels = AccountLabels | WalletLabels;
+export type Labels = (AccountLabels | WalletLabels) & {
+    /**
+     * in case file was migrated from a different file. reference to previous file
+     */
+    migratedFrom?: string;
+    /**
+     * we might create some dummy data to make it hard to guess number of labels by file size
+     */
+    dummy?: string;
```

Comment:
should we also bump `version` field saved in file ?

Reply:
might make sense to keep track of this [dac3416](https://github.com/trezor/trezor-suite/pull/9130/commits/dac34163913bc2dabb816d4aa46213ab23a617a3)